doc/2/guides/develop-on-kuzzle/event-system/index.md

```diff
@@ -187,3 +187,25 @@ await app.trigger('app-name/file-available', fileUrl);
 ::: warning
 If an internal event is triggered, the payload must be the same as the original event.
 :::
+
+**Events can be triggered** with SDK and controllers actions.
```

Comment:
```suggestion
**Events are always triggered** from SDK and controller actions.
```

Reply:
But since the default behavior without the flag is to not trigger events wouldn't it be confusing ?

Reply:
Well they are triggered from external SDK and controller actions, so I think that's a bit misleading

Reply:
True...
What about :
**Events are not triggered** with the embedded SDK and controllers actions.

::: warning
By default, actions executed through the embedded SDK or controllers won't trigger any events and thus no pipe or hooks will be called. On the contrary, controllers accessed by the external SDK through HTTP or Websocket requests will always fire events.
:::